### PR TITLE
feat:gyroscope add moveMode option

### PR DESCRIPTION
**Pull request by Cansiny0320**
**Merge request checklist**

- [x] I read the [guidelines for contributing](https://github.com/mistic100/Photo-Sphere-Viewer/blob/master/.github/CONTRIBUTING.md)
- [x] I created my branch from `dev` and I am issuing the PR to `dev`
- [x] Unit tests are OK

### CODE REVIEW

**Comment by Cansiny0320** [comment]
close #693

File: docs/plugins/plugin-gyroscope.md [hunk 1]
```diff
@@ -38,6 +38,12 @@ Allows to pan horizontally the camera when the gyroscope is enabled (requires gl
 
 By default the camera will keep its current horizontal position when the gyroscope is enabled. Turn this option `true` to enable absolute positionning and only use the device orientation.
 
+#### `moveMode`
+- type: `smooth` | `fast`
+- default: `smooth`
+
+move smooth or fast
```
Comment (mistic100) [inline docs/plugins/plugin-gyroscope.md:45]:
```suggestion
How the gyroscope data is used to rotate the panorama.
```

File: src/plugins/gyroscope/index.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ import { GyroscopeButton } from './GyroscopeButton';
  * @typedef {Object} PSV.plugins.GyroscopePlugin.Options
  * @property {boolean} [touchmove=true] - allows to pan horizontally when the gyroscope is enabled (requires global `mousemove=true`)
  * @property {boolean} [absolutePosition=false] - when true the view will ignore the current direction when enabling gyroscope control
+ * @property {'smooth' | 'fast'} [moveMode='smooth'] - move smooth or fast
```
Comment (mistic100) [inline src/plugins/gyroscope/index.js:12]:
```suggestion
 * @property {'smooth' | 'fast'} [moveMode='smooth'] - How the gyroscope data is used to rotate the panorama
```

**Review by mistic100 — CHANGES_REQUESTED** [review]
Missing the change in "types"

**Comment by Cansiny0320** [comment]
What else needs to be modified? @mistic100

**Comment by mistic100** [comment]
> What else needs to be modified? @mistic100

my previous comment https://github.com/mistic100/Photo-Sphere-Viewer/pull/695#pullrequestreview-1005676245

**Comment by Cansiny0320** [comment]
> 
Maybe I understand the wrong, do I want me to modify the explanation of the type? I have modified it

**Comment by Cansiny0320** [comment]
> > What else needs to be modified? @mistic100
> 
> my previous comment [#695 (review)](https://github.com/mistic100/Photo-Sphere-Viewer/pull/695#pullrequestreview-1005676245)

Maybe I understand the wrong, do I want me to modify the explanation of the type? I have modified it

**Comment by Cansiny0320** [comment]
> types

oh,i got it

**Comment by Cansiny0320** [comment]
> > What else needs to be modified? @mistic100
> 
> my previous comment [#695 (review)](https://github.com/mistic100/Photo-Sphere-Viewer/pull/695#pullrequestreview-1005676245)

oh,i got it

**Pull request merged**